README.adoc

```diff
@@ -1,50 +1,111 @@
-
-Follow steps found in the `Building Projects with Native Code` tab in the link:https://facebook.github.io/react-native/docs/getting-started.html[React Native documentation for Getting Started] for installing dependencies and setting up the mobile development environment, such as the Android SDK and build tools.
-
-=== Setting up a Device
-
-To try running the code on a device, follow the steps found in the link:https://facebook.github.io/react-native/docs/running-on-device.html[React Native documentation for Running your app on a Device].
-
-==== Common Issues
-
-* Make sure the `JAVA_HOME` and `ANDROID_HOME` environment variables are set.
-
-* Make sure the `adb` command is available from the terminal.
-The sdk platform tools directory may have to be added to the `PATH` environment variables.
-The platform tools directory can be found in `D:\Users\<your username>\AppData\Local\Android\sdk\platform-tools`.
-
-* When trying to run the application on a device, make sure a phone is connected.
-Use `adb devices` to check for a connected device.
-
-
-=== Running the Application
-
-Once a development environment has been setup, the application can be run with the following steps.
-
-. Clone the repository.
-. Run `yarn install` to install the dependencies.
-. Ensure a device is connected to the computer or an emulator is set up.
-. Run `npm run start:android` or `npm run start:ios` to run the application.
-
-=== Running Tests
-
-. Run `npm test` to execute all unit tests and check code coverage thresholds.
-. View code coverage in the local `coverage/` directory.
-
-=== Contributing
-
-Please label branches using the following format:
-
-	<full name>/AT-<issue number>/<feature name>
-
-
-Please write commit messages using the following format:
-
-	AT-<issue number> <commit message>
+= agritrader
+
+This project is a mobile application to be used by traders that interact with farmers to collect agricultural goods.
+
+== Development
+
+=== Setting up a Development Environment via Docker
+
+A Docker development environment has been made available, to use it follow the steps below.
+
+IMPORTANT: This does not work on native docker for Windows and OS X.
+This works natively on Linux.
+It can work on Docker Toolbox by link:https://gist.github.com/stonehippo/e33750f185806924f1254349ea1a4e68[passing the USB device through to VirtualBox] first.
+
+. Install Docker using the instructions on the link:https://docs.docker.com/engine/installation/[official website].
+Do not use a package manager as you will get an outdated version of Docker.
+
+. Give executable permissions to the utility script using `chmod +x dockerize.sh`.
+
+. Build the image using `./dockerize.sh init`.
+This may take a while.
+
+. Install the node modules using `./dockerize.sh install`.
+
+. Plug in your phone to your computer and allow access to your phone.
+Allowing access to your phone varies on the phone but is commonly a pop-up shortly after plugging in a phone to a computer.
```

Comment:
The flow of this sentence is a bit odd. Could it be reworded? Maybe to something like this:
Allowing access to your phone varies on the phone, but it is common for a pop-up to show on your device after plugging it into a computer.